src/components/Modals/FiatRamps/views/RampsList.tsx

```diff
@@ -16,79 +17,97 @@ type RampsListProps = {
 
 export const RampsList: React.FC<RampsListProps> = ({ setFiatRampProvider }) => {
   const history = useHistory()
+  const coinbasePayFeatureFlag = useFeatureFlag('CoinbasePay')
 
   const ramps = useMemo(() => {
     type Entry = [keyof typeof supportedFiatRamps, SupportedFiatRampConfig]
     const initial: ReactElement[] = []
-    const result = (Object.entries(supportedFiatRamps) as Entry[]).reduce((acc, entry) => {
-      const [fiatRamp, fiatRampConfig] = entry
-      if (fiatRampConfig.isImplemented) {
-        acc.unshift(
-          <Button
-            key={fiatRamp}
-            width='full'
-            height='auto'
-            justifyContent='space-between'
-            variant='ghost'
-            fontWeight='normal'
-            py={2}
-            onClick={() => {
-              setFiatRampProvider(fiatRamp)
-              history.push(FiatRampsRoutes.Manager)
-            }}
-            rightIcon={<ChevronRightIcon boxSize={6} />}
-          >
-            <Flex
-              flex={1}
-              flexDirection={['column', 'row']}
+    const result = (Object.entries(supportedFiatRamps) as Entry[]).reduce(
+      (acc, supportedFiatRamp) => {
+        const [fiatRamp, fiatRampConfig] = supportedFiatRamp
+        if (fiatRamp === 'CoinbasePay' && !coinbasePayFeatureFlag) {
+          // Since isImplemented is set in a config file where we can't use hooks,
+          // we reassign isImplemented here based on the feature flag.
+          fiatRampConfig.isImplemented = false
+        }
```

Comment:
```suggestion
        if (fiatRamp === 'CoinbasePay') {
          // Since isImplemented is set in a config file where we can't use hooks,
          // we reassign isImplemented here based on the feature flag.
          fiatRampConfig.isImplemented = coinbasePayFeatureFlag
        }
```

Reply:
@gomesalexandre Done.